### DISCUSSION TITLE
Issue: Randomly the maximum volume from MPV becomes much lower than normal

### DISCUSSION THREAD

**Discussion by lamyergeier** [mpv-player/mpv#14004] [Q&A]
EDIT:

Please see: [[Bug Randomly the maximum volume from MPV becomes much lower than normal · Issue #14008 · mpv-player/mpv](https://github.com/mpv-player/mpv/issues/14008)](https://github.com/mpv-player/mpv/issues/14008)

---

I am facing this  trouble with mpv that randomly the maximum volume from MPV becomes much lower than normal. It stays like that for random duration and then randomly it becomes fine for unknown duration. I face this problem daily. 

```
$ mpv --version
mpv v0.38.0 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
libplacebo version: v6.338.2
FFmpeg version: n7.0
FFmpeg library versions:
   libavutil       59.8.100
   libavcodec      61.3.100
   libavformat     61.1.100
   libswscale      8.1.100
   libavfilter     10.1.100
   libswresample   5.1.100
```

I was facing this trouble with mpv 0.37 and was hoping that the issue might go away with the upgrade to 0.38 but it is still there.

**Comment by haasn** [comment]
Are you by any chance using PulseAudio?

**Reply by lamyergeier** [reply]
@haasn Please check:[Bug Randomly the maximum volume from MPV becomes much lower than normal · Issue #14008 · mpv-player/mpv](https://github.com/mpv-player/mpv/issues/14008)

I am using pipewire 1.0.5
